cmd/cli.go

```diff
@@ -23,6 +30,56 @@ func newCli(GitCommit string, GitDate string) *cli.App {
 				Description: "Display the current mocktimism config",
 				Action:      actionConfig,
 			},
+			{
+				Name:        "anvil",
+				Flags:       configFlags,
+				Description: "Starts the anvil services",
+				Action: func(ctx *cli.Context) error {
+					// TODO extract the code to run every process into a reusable function https://github.com/ethereum-optimism/mocktimism/issues/73
+					var wg sync.WaitGroup
+					errCh := make(chan error, 5)
+
+					log := oplog.NewLogger(oplog.AppOut(ctx), oplog.ReadCLIConfig(ctx)).New("role", "mocktimism")
+					oplog.SetGlobalLogHandler(log.GetHandler())
+					cfg, err := config.LoadNewConfig(log, ctx.String(ConfigFlag.Name))
+					if err != nil {
+						log.Error("failed to load config", "err", err)
+						return err
+					}
+
+					processCtx, processCancel := context.WithCancel(ctx.Context)
+
+					runService := func(start func(ctx context.Context) error) {
+						wg.Add(1)
+						go func() {
```

Comment:
If you're interested, @protolambda has worked on some lifecycle improvements for the indexer you can leverage here to manage resources more cleanly

https://github.com/ethereum-optimism/optimism/pull/7961

Reply:
Definitely interested! Made a ticket https://github.com/ethereum-optimism/mocktimism/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=